docset/winserver2012-ps/netadapter/Get-NetAdapter.md

```diff
@@ -16,126 +16,127 @@ Gets the basic network adapter properties.
 
 ## SYNTAX
 
-### UNNAMED_PARAMETER_SET_1
-```
+### ByName (Default)
+```powershell
 Get-NetAdapter [[-Name] <String[]>] [-AsJob] [-CimSession <CimSession[]>] [-IncludeHidden] [-Physical]
  [-ThrottleLimit <Int32>]
 ```
 
-### UNNAMED_PARAMETER_SET_2
-```
+### ByInstanceID
+```powershell
 Get-NetAdapter [-AsJob] [-CimSession <CimSession[]>] [-IncludeHidden] [-Physical] [-ThrottleLimit <Int32>]
  -InterfaceDescription <String[]>
 ```
 
-### UNNAMED_PARAMETER_SET_3
-```
+### ByIfIndex
+```powershell
 Get-NetAdapter [-AsJob] [-CimSession <CimSession[]>] [-IncludeHidden] [-Physical] [-ThrottleLimit <Int32>]
  -InterfaceIndex <UInt32[]>
 ```
 
 ## DESCRIPTION
 The **Get-NetAdapter** cmdlet gets the basic network adapter properties.
 By default only visible adapters are returned.
-To see the common network adapter properties, pipe the output into the Format-Listhttp://go.microsoft.com/fwlink/p/?LinkID=113302 cmdlet.
-To see all the properties, pipe the output to the Format-Listhttp://go.microsoft.com/fwlink/p/?LinkID=113302 with the **Property** parameter specified as any (`*`).
+To see the common network adapter properties, pipe the output into the **Format-List** cmdlet.
+To see all the properties, pipe the output to the **Format-List** cmdlet with the *Property* parameter specified as the wildcard character "*".
 This cmdlet supports multiple views.
 The default view is as a table.
-To see more information regarding various network adapter identifiers use the names view using the Format-Tablehttp://go.microsoft.com/fwlink/p/?LinkID=113303 cmdlet with the **View** parameter specified as name.
-To see more information regarding the miniport (device driver) such as driver date or version use the driver view using the Format-Tablehttp://go.microsoft.com/fwlink/p/?LinkID=113303 cmdlet with the **View** parameter specified as driver.
+To see more information regarding various network adapter identifiers use the **names** view using the **Format-Table** cmdlet with the *View* parameter specified as **name**.
+To see more information regarding the miniport, device driver, such as driver date or version use the **driver** view using the **Format-Table** cmdlet with the *View* parameter specified as **driver**.
 
 ## EXAMPLES
 
-### EXAMPLE 1
-```
-PS C:\>Get-NetAdapter -Name *
+### Example 1: Get all visible network adapters
+```powershell
+PS C:\> Get-NetAdapter -Name "*"
```

Comment:
I'm confused as to why the quotes are around the asterisk. This is unnecessary and just adds more characters to type. In general, this is true of _most_ cmdlets that accept string input.

Reply:
I have just made example across article versions the same. I agree with you and it is probably better idea to copy the 2012 example to the other ones.

Reply:
Yes, it works with our without the quotes.